src/vendor/bs58.js

```diff
@@ -9,15 +9,17 @@
 
 var ALPHABET = 'gsphnaf39wBUDNEGHJKLM4PQRST7VWXYZ2bcdeCr65jkm8oFqi1tuvAxyz';
 var ALPHABET_MAP = {};
-for(var i = 0; i < ALPHABET.length; ++i) {
+for (var i = 0; i < ALPHABET.length; ++i) {
   ALPHABET_MAP[ALPHABET.charAt(i)] = i;
 }
 var BASE = 58;
 
 function decode(string) {
   if (string.length === 0) return [];
 
-  var i, j, bytes = [0];
+  var i,
```

Comment:
This formatting change is funky. Are you sure it's better?

Reply:
Good call, that should just be ignored